docs/Hardlinks/How-to-setup-for/Docker.md

```diff
@@ -2,23 +2,23 @@
 
 !!! tip
 
-    If you're new to dockers and want a easy setup I suggest to take a look at [DockSTARTer](https://dockstarter.com/){:target="_blank" rel="noopener noreferrer"}.
+    If you're new to Docker and want an easy setup, I suggest looking at [DockSTARTer](https://dockstarter.com/){:target="_blank" rel="noopener noreferrer"}.
     I've also created a short guide [HERE](/Hardlinks/How-to-setup-for/Dockstarter/) where I explain the settings for the most used applications.
 
     The main goal of DockSTARTer is to make it quick and easy to get up and running with Docker.
     You may choose to rely on DockSTARTer for various changes to your Docker system or use DockSTARTer as a stepping stone and learn to do more advanced configurations.
 
-    *DockSTARTer was actually my first steps in to the world of dockers.*
+    *DockSTARTer was actually my first steps into the world of Docker.*
 
-!!! note
+!!! notes
```

Comment:
@TRaSH- is this a specific keyword at all?

Reply:
https://squidfunk.github.io/mkdocs-material/reference/admonitions/